packages/app/src/lib/auction/useAuctionStart.ts

```diff
@@ -179,12 +186,50 @@ export function useAuctionStart() {
         lastAuctionRef.current = params;
         setCurrentAuctionParams(params);
 
+        // Generate SIWE signature for the auction request
+        let takerSignature: string | undefined;
+        let takerSignedAt: string | undefined;
+        try {
+          const { domain, uri } = extractSiweDomainAndUri(wsUrl);
+          const issuedAt = new Date().toISOString();
+          const signingPayload: AuctionStartSigningPayload = {
+            wager: params.wager,
+            predictedOutcomes: params.predictedOutcomes,
+            resolver: params.resolver,
+            taker: params.taker,
+            takerNonce: params.takerNonce,
+            chainId: params.chainId,
+          };
+          const message = createAuctionStartSiweMessage(
+            signingPayload,
+            domain,
+            uri,
+            issuedAt
+          );
+          takerSignature = await signMessageAsync({ message });
+          takerSignedAt = issuedAt;
+          if (process.env.NODE_ENV !== 'production') {
+            console.debug('[AuctionStart] Generated SIWE signature');
+          }
+        } catch (signError) {
+          // If signature fails, log but continue without it (optional signature)
+          console.warn(
+            '[AuctionStart] Failed to sign auction request:',
+            signError
+          );
+        }
+
+        // Add signature and timestamp to request payload if available
+        const payloadWithSignature = takerSignature && takerSignedAt
```

Comment:
meaning we always send with signature unless failed? could you make a case when we need to send a request without a signature at all?